### PR TITLE
[NO ISSUE] feat: change fontSize of drawer heading

### PR DESCRIPTION
**Pull request by helfer1991**
## Description of the change

Goal of the PR is to make the Drawer component more flexible, allowing, for example, the title/heading to have a marginBottom and to fix its font-size to match the designs for the Partners user story of the TREX team
## Testing the change

- [ ] Write your testing instructions here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Non-Breaking Change (change to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.

### CODE REVIEW

File: packages/components/src/components/Drawer/DrawerHeader.tsx [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
+    { children, marginBottom = "space0", padding = "space60", ...props },
+    ref,
+  ) => {
     return (
       <Box.div
+        alignItems="center"
         display="flex"
         gap="space30"
         justifyContent="space-between"
+        marginBottom={marginBottom}
```
Comment (apvale) [inline packages/components/src/components/Drawer/DrawerHeader.tsx:27]: Why do we need a marginBottom? I ran storybook and noticed that it's showing ok for what we need, no? I mean, padding should be enough. Also, we don't set marginBottom on Figma.

image

File: packages/components/src/components/Heading/Heading.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import forEach from "lodash/forEach";
 import { Text } from "../../primitives/Text";
 
 type HeadingLevelOptions = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
-type HeadingMarginOptions = "space0" | "space70";
+type HeadingMarginOptions = SystemProp<keyof Theme["space"], Theme>;
```
Comment (apvale) [inline packages/components/src/components/Heading/Heading.tsx:9]: Why did you change this?

Comment (helfer1991) [reply packages/components/src/components/Heading/Heading.tsx:9]: well, because instead of `"space0" | "space60" | "space160" | ...` I think it's more appropriate to make this more flexible and able to receive any `spaceSuffix` available

File: packages/components/src/components/Drawer/DrawerHeader.tsx (hunk 1 above)
Comment (helfer1991) [reply packages/components/src/components/Drawer/DrawerHeader.tsx:27]: I get your point completely, this is just to provide more flexibility to this component for future use.
If you think this specific change should be deleted I'm fine with it, just let me know.

File: packages/components/src/components/Heading/Heading.tsx (hunk 2 above)
Comment (apvale) [reply packages/components/src/components/Heading/Heading.tsx:9]: I understand the reason to change this, but, IMHO, it shouldn't be part of this PR. This involves changing the Heading component that's not even being mentioned in the changeset, for example.

File: packages/components/src/components/Drawer/DrawerHeader.tsx (hunk 1 above)
Comment (apvale) [reply packages/components/src/components/Drawer/DrawerHeader.tsx:27]: For me, with the changes you did on this [PR](https://github.com/Localitos/pluto/pull/1490), we already have a padding of 16px that it's the same what we have on Figma. So, it seems to me that we don't need to introduce anything related to spaces on this PR.

File: packages/components/src/components/Heading/Heading.tsx (hunk 2 above)
Comment (helfer1991) [reply packages/components/src/components/Heading/Heading.tsx]: yeah, you're right. reverted those changes

File: .changeset/wise-feet-clean.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": minor
+---
+
+enable margin bottom to the title and change its font size
```
Comment (apvale) [inline .changeset/wise-feet-clean.md:5]: I think we just want to mention that we update the fontSize, right?

**Review by apvale — APPROVED** [review]

File: .changeset/wise-feet-clean.md (hunk 3 above)
Comment (helfer1991) [reply .changeset/wise-feet-clean.md:5]: yes, forgot about this, good catch

Comment (helfer1991) [reply .changeset/wise-feet-clean.md]: done
